flink-connectors/flink-connector-aws-kinesis-data-streams/src/main/java/org/apache/flink/connector/kinesis/sink/KinesisDataStreamsSinkElementConverter.java

```diff
@@ -54,7 +54,7 @@ private KinesisDataStreamsSinkElementConverter(
         this.partitionKeyGenerator = partitionKeyGenerator;
     }
 
-    @Experimental
+    @Internal
```

Comment:
Same question as above, why do we need to annotate these methods?

Reply:
Because otherwise I get `org.apache.flink.connector.kinesis.sink.KinesisDataStreamsSinkElementConverter.builder(): Returned leaf type org.apache.flink.connector.kinesis.sink.KinesisDataStreamsSinkElementConverter$Builder does not satisfy`.

I cant seem to find a solution, will keep trying

Reply:
I guess I can make the `ElementConverter` `@Internal`....
But it's still the case that @vahmed-hamdy is using some parts for public parts and the converter is an internal tool.
```
KinesisDataStreamsSinkElementConverter.apply(java.lang.Object, org.apache.flink.api.connector.sink.SinkWriter$Context): Argument leaf type org.apache.flink.api.connector.sink.SinkWriter$Context does not satisfy: reside outside of package 'org.apache.flink..' or annotated with @Public or annotated with @PublicEvolving or annotated with @Deprecated
KinesisDataStreamsSinkElementConverter.apply(java.lang.Object, org.apache.flink.api.connector.sink.SinkWriter$Context): Argument leaf type org.apache.flink.api.connector.sink.SinkWriter$Context does not satisfy: reside outside of package 'org.apache.flink..' or annotated with @Public or annotated with @PublicEvolving or annotated with @Deprecated
```

Reply:
As discussed, this class should be `@Internal`